CMakeLists.txt

```diff
@@ -221,27 +226,29 @@ set(NMC_VALIDATION_DATA_DIR "${PROJECT_SOURCE_DIR}/validation/data" CACHE PATH
   "location of generated validation data")
 
 #----------------------------------------------------------
-# Whether to build validation data at all
+# Whether to build validation data
```

Comment:
How big is the validation data? Would it be possible to have a backup pregenerated set available on the repo?